projects/igniteui-angular/src/lib/core/styles/components/grid/_grid-theme.scss

```
@@ -1371,6 +1381,9 @@
         }
     }
 
+    %grid-cell--pinned-chip {
+        margin-right: rem(4px);
```

```suggestion
        margin-#{$right}: rem(4px);
```
Where did you get the 4px from?

---

Based on the spec screenshots we have, this can easily be changed if you think pixels should not be used or anything.
Maybe @StefanIvanov could also say what should be the correct value.

---

Yes we keep 4px between the chip and the cell text, not sure if the value should depend on display density but maybe we can use a fraction of the cell left/right padding.

---

We updated the spec with the margins for the different densities, thanks for noting this.